doc/admin-tasks.md

```diff
@@ -0,0 +1,11 @@
+## Admin Tasks
```

Comment:
Thanks for writing this up as documentation 👍  I'm wondering if we actually need it though - as it doesn't really tell you any more than running `rake -T` it might be prudent to just suggest running that in the main readme rather than linking to this file? A concern of mine is that we'd want this file to have all the rake tasks listed pretty much and then we might just be duplicating (and potentially) falling out of sync with the rake documentation.